bpf/bpf_sock.c

```diff
@@ -60,6 +60,19 @@ ctx_src_port(const struct bpf_sock *ctx)
 	return (__be16)bpf_htons(sport);
```

Comment:
For the commit description and history:

I'd recommend you squash the two commits together and change the title to something short (<60 characters) along the lines of `bpf: Remove reverse NAT entries on sock_release`. You'll also need to remove the merge commit and rebase on main instead. Note you can force push to your branch to update it; that's not an issue.

Then, the commit description needs to provide the reason for these changes, the information that is currently in your pull request description. Don't hesitate to look at examples in the repository for the files you touched to get an idea of what's expected.